assets/js/base/context/hooks/cart/test/use-store-cart.js

```diff
@@ -165,14 +165,17 @@ describe( 'useStoreCart', () => {
 		} );
 
 		it( 'return default data when shouldSelect is false', () => {
-			const TestComponent = getTestComponent( { shouldSelect: false } );
+			const TestComponent = getTestComponent( {
+				shouldSelect: false,
+			} );
 
 			act( () => {
 				renderer = TestRenderer.create(
 					getWrappedComponents( TestComponent )
 				);
 			} );
 
+			//eslint-disable-next-line testing-library/await-async-query
```

Comment:
I propose to make a ticket to treat/replace these comments in the future, because at the end of the day, it is about using the [Testing library's API wrong](https://github.com/testing-library/eslint-plugin-testing-library/blob/main/docs/rules/await-async-query.md).

Reply:
In this case, it seems eslint is incorrectly applying these rules that should be for `testing-library/react` to the `react-teset-renderer` library, which from what I can tell is different. So i think these comments are legit, but I didn't look into why eslint is doing this